generated/nidcpower/nidcpower/session.py

```diff
@@ -5703,6 +5703,61 @@ def _parse_channel_count(self):
+
+        You must physically configure your LCR circuit with an appropriate reference load to use this method to generate valid load compensation data.
+
+        When you call this method:
+
+        -  The load compensation data is written to the onboard storage of the instrument. Onboard storage can contain only the most recent set of data.
+        -  Most NI-DCPower properties in the session are reset to their default values. Rewrite the values of any properties you want to maintain.
+
+        To apply the load compensation data you generate with this method to your LCR measurements, set the lcr_load_compensation_enabled property to True.
+
+        Load compensation data are generated only for those specific frequencies you define with this method; load compensation is not interpolated from the specific frequencies you define and applied to other frequencies.
+
+        Note:
+        This method is not supported on all devices. For more information about supported devices, search ni.com for Supported Methods by Device.
+
+        Tip:
+        This method can be called on specific channels within your :py:class:`nidcpower.Session` instance.
+        Use Python index notation on the repeated capabilities container channels to specify a subset,
+        and then call this method on the result.
+
+        Example: :py:meth:`my_session.channels[ ... ].perform_lcr_load_compensation`
+
+        To call the method on all channels, you can call it directly on the :py:class:`nidcpower.Session`.
+
+        Example: :py:meth:`my_session.perform_lcr_load_compensation`
+
+        Args:
+            compensation_spots (list of LCRLoadCompensationSpot): Defines the frequencies and DUT specifications to use for LCR load compensation.
+
+                You can specify <=1000 spot frequencies.
+
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | frequency            | The spot frequency.                                                                                                                                   |
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | reference_value_type | A known specification value of your DUT to use as the basis for load compensation. Refer to the enums.LCRReferenceValueType for the supported values. |
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | reference_value_A    | A value that describes the **reference_value_type** specification. Use as indicated by the **reference_value_type** option you choose.                |
```

Comment:
There is no ref value A and B in the public API. I'd suggest making this piece of documentation be identical to the docstring in LCRLoadCompensationSpot constructor.

Reply:
As mentioned in https://github.com/ni/nimi-python/pull/1772/files/6dec3e3077ca16f01cb4ba7b856d40cb47f1759c#r924209556, this docstring will be fixed after including the metadata update from #1771

Reply:
This docstring has been fixed in the latest merge commit.